### ISSUE TITLE
[BUG 🪲] Error while installing with pip

### ISSUE THREAD

**Issue by andrefpoliveira** [NiiightmareXD/windows-capture#4]
**Describe the bug 🐛**
While trying to install the library with `pip install windows-capture`, I got the following error:
```
C:\Users\Andre.LAPTOP01>pip install windows-capture
WARNING: Ignoring invalid distribution -illow (c:\users\andre.laptop01\appdata\local\programs\python\python310\lib\site-packages)
Collecting windows-capture
  Using cached windows_capture-1.0.42.tar.gz (33 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [10 lines of output]
      error: failed to parse lock file at: C:\Users\Andre.LAPTOP01\AppData\Local\Temp\pip-install-ufh0qrqo\windows-capture_da4973df594a487dacd1b381176f7e6a\windows-capture-python\Cargo.lock

      Caused by:
        package `windows-capture` is specified twice in the lockfile
      ðŸ’¥ maturin failed
        Caused by: Cargo metadata failed. Does your crate compile with `cargo build`?
        Caused by: `cargo metadata` exited with an error:
      Error running maturin: Command '['maturin', 'pep517', 'write-dist-info', '--metadata-directory', 'C:\\Users\\Andre.LAPTOP01\\AppData\\Local\\Temp\\pip-modern-metadata-vd2ex463', '--interpreter', 'C:\\Users\\Andre.LAPTOP01\\AppData\\Local\\Programs\\Python\\Python310\\python.exe']' returned non-zero exit status 1.
      Checking for Rust toolchain....
      Running `maturin pep517 write-dist-info --metadata-directory C:\Users\Andre.LAPTOP01\AppData\Local\Temp\pip-modern-metadata-vd2ex463 --interpreter C:\Users\Andre.LAPTOP01\AppData\Local\Programs\Python\Python310\python.exe`
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Expected behavior 📝**
Instalation successful

**OS 🤖**
 - Version: [e.g. 11]
 - Build[e.g. 22H2]

**Additional context ➕**

**Comment by NiiightmareXD** [comment]
should be fixed in v1.0.45

**Comment by NiiightmareXD** [comment]
Thanks for reporting the bug, it should be fixed in v1.0.45 try it out 😊.

**Comment by andrefpoliveira** [comment]
Thanks! The installation was successful. However, the base code in the repo:
```
from windows_capture import WindowsCapture, Frame, InternalCaptureControl

# Every Error From on_closed and on_frame_arrived Will End Up Here
capture = WindowsCapture(
    capture_cursor=None,
    draw_border=None,
    monitor_index=None,
    window_name=None,
)


# Called Every Time A New Frame Is Available
@capture.event
def on_frame_arrived(frame: Frame, capture_control: InternalCaptureControl):
    print("New Frame Arrived")

    # Save The Frame As An Image To The Specified Path
    frame.save_as_image("image.png")

    # Gracefully Stop The Capture Thread
    capture_control.stop()


# Called When The Capture Item Closes Usually When The Window Closes, Capture
# Session Will End After This Function Ends
@capture.event
def on_closed():
    print("Capture Session Closed")


capture.start()
```

Returns the following error:
```
Traceback (most recent call last):
  File "c:\Users\user\Desktop\test.py", line 31, in <module>
    capture.start()
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\windows_capture\__init__.py", line 204, in start
    self.capture.start()
Exception: Capture Session Threw An Exception -> TypeError: can't multiply sequence by non-int of type 'float'
```

**Comment by NiiightmareXD** [comment]
Thanks for reporting, this one should be fixed in v1.0.50.

**Comment by andrefpoliveira** [comment]
Working! 😄
